style/app.scss

```diff
@@ -2487,14 +2480,44 @@ main-filter input
 
 .tooltip-group-display
 {
-  cursor: pointer;
-  transition: opacity 0.4s linear;
-  opacity:1;
+  cursor     : pointer;
+  transition : opacity 0.4s linear;
+  opacity    : 1;
 }
 
 .tooltip-group-hidden
 {
-  cursor: pointer;
-  transition: opacity 0.4s linear;
-  opacity:0;
+  cursor     : pointer;
+  transition : opacity 0.4s linear;
+  opacity    : 0;
+}
+
+.graph-dropdown-hidden
+{
+  opacity    : 0;
+  position   : absolute;
+  display    : inline-block;
+  height     : 0;
+  overflow   : hidden;
+  transition : opacity 0.4s linear;
 }
+.graph-dropdown-display
+{
+  background-color : #8a75ae;
+  font-size        : 15px;
+  display          : inline-block;
+  opacity          : 1;
+  transition       : opacity 0.4s linear;
+  list-style       : none;
+  padding-left     : 20px;
+  column-count: 3;
+  text-align: left;
+  border-bottom: 10px solid #8a75ae;
+  position:absolute;
+}
+.graph-dropdown-display h4
+{
+  font-weight: bold;
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
```

Comment:
Make sure this file ends in a blank line. In fact, you should configure your text editor to always do this automatically.